src/state/store.ts

```diff
@@ -28,7 +28,7 @@ import { updateWindowStoreMiddleware } from './windowMiddleware'
 
 const persistConfig = {
   key: 'root',
-  version: 1,
+  version: 3,
```

Comment:
crux of the fix, we weren't running migrations since version 1 💀
I've removed all of the ones that weren't run so we arent invalidating cache the doesnt need to be.

Reply:
!!!! this explains how why we see so many errors in mixpanel

every cache migration wasn't running, users were clicking the uh oh something went wrong, which would clear the cache, getting them back to a good state

even better than this, can we use `version = Math.max(...Object.keys(migrations).map(Number))` so it always runs the latest?